ext/filter/tests/030.phpt

```diff
@@ -51,7 +51,7 @@ $ipv6_test = array(
 );
 foreach ($ipv6_test as $ip => $exp) {
 	$out = filter_var($ip, FILTER_VALIDATE_IP, FILTER_FLAG_IPV6);
-	$out = (int) ($out === false ? 0 : 1);
+	$out = (int) ($out !== false);
```

Comment:
The (int) cast here looks fishy. Looking at the surrounding code, I think just keeping the boolean here would be fine.